app/components/Scenes/Badge.jsx

```diff
@@ -26,10 +26,10 @@ import { MeshLineGeometry, MeshLineMaterial } from "meshline";
 
 extend({ MeshLineGeometry, MeshLineMaterial });
 
-export default function BadgeScene() {
+export default function Badge() {
   // const { debug } = useControls({ debug: false });
   return (
-    <Canvas camera={{ position: [0, 0, 13], fov: 25 }}>
+    <Canvas camera={{ position: [0, 0, 13], fov: 25 }} className="bg-black">
       <ambientLight intensity={Math.PI} />
```

Comment:
Something is wrong with the band texture and it's clipping the clip. This needs to be fixed

Reply:
Referenced in Issue #57